backend/main.go

```diff
@@ -14,10 +14,16 @@ func main() {
 	http.Handle("/", r)
 
 	// test route to add data to firestore
-	r.HandleFunc("/add", AddData).Methods("POST")
-	r.HandleFunc("/get/{id}", GetData).Methods("GET")
-	r.HandleFunc("/delete/{id}", DeleteData).Methods("DELETE")
-	r.HandleFunc("/update/{id}", UpdateUser).Methods("PATCH")
+	r.HandleFunc("/users", CreateUser).Methods("POST")
+	r.HandleFunc("/users/{id}", GetUser).Methods("GET")
+	r.HandleFunc("/users/{id}", DeleteUser).Methods("DELETE")
+	r.HandleFunc("/users/{id}", UpdateUser).Methods("PATCH")
+
+	//Event endpoints
+	r.HandleFunc("/events",CreateEvent).Methods("POST")
```

Comment:
nit: space between comma and argument

Reply:
@jackyzha0 would you wanna set up a linter for Go? I think you mentioned this a few weeks ago